cypress/integration/pages/articlesWithAv/index.js

```diff
@@ -0,0 +1,31 @@
+// Based on https://github.com/cypress-io/cypress/issues/136#issuecomment-342391119
+// TODO benefits of consider alternate approaches on same issue
+Cypress.Commands.add('iframeBody', { prevSubject: 'element' }, $iframe => {
+  return new Cypress.Promise(resolve => {
+    $iframe.on('load', () => {
+      resolve($iframe.contents().find('body'));
+    });
+  });
+});
+
+describe('Audio/video playback', () => {
+  const audioVideoUrl = 'https://www.test.bbc.co.uk/news/articles/cn7k01xp8kxo';
+
+  it('includes guidance message and subtitles', () => {
+    cy.visit(audioVideoUrl);
+    cy.get('div[class^=StyledVideoContainer]').click();
+
+    cy.get('iframe[class^="StyledIframe"]')
+      .iframeBody()
+      .find('iframe#smphtml5iframemediaPlayer')
```

Comment:
Would it be safer to just use the first `iframe` in the `StyledIframe`? Theres only 1 atm and it seems less likely that they will add a second before the main one, than them changing their class name

Reply:
This is fine provided one piece of AV content is sufficient to test all our important use cases, which I'm not sure it is.